test/react-web/client/graphql/queries.test.tsx

```diff
@@ -2113,4 +2113,39 @@ describe('queries', () => {
     );
   });
 
+  it('uses a custom wrapped component name when \'alias\' is specified', (done) => {
+    const query = gql`query people { allPeople(first: 1) { people { name } } }`;
+    const data = { allPeople: { people: [ { name: 'Luke Skywalker' } ] } };
+    const networkInterface = mockNetworkInterface({ request: { query }, result: { data } });
+    const client = new ApolloClient({ networkInterface, addTypename: false });
+
+    let queryExecuted;
+    @graphql(query, {
+      alias: 'withFoo',
+    })
+    class Container extends React.Component<any, any> {
+      componentWillReceiveProps(props) {
```

Comment:
I don't think you need to do all this; it's sort of testing the `query.metadata` feature is working but there's a separate test for that (I assume).

Can you not just do 

```js
const instance = renderer.create(<Container>);
instance.displayName;
```

?